packages/rpc-core/src/rpc-methods/__tests__/get-first-available-block-test.ts

```diff
@@ -0,0 +1,23 @@
+import { createHttpTransport, createJsonRpc } from '@solana/rpc-transport';
+import type { Rpc } from '@solana/rpc-transport/dist/types/json-rpc-types';
+import fetchMock from 'jest-fetch-mock-fork';
+import { SolanaRpcMethods, createSolanaRpcApi } from '../../index';
+
+describe('getFirstAvailableBlock', () => {
+    let rpc: Rpc<SolanaRpcMethods>;
+    beforeEach(() => {
+        fetchMock.resetMocks();
+        fetchMock.dontMock();
+        rpc = createJsonRpc<SolanaRpcMethods>({
+            api: createSolanaRpcApi(),
+            transport: createHttpTransport({ url: 'http://127.0.0.1:8899' }),
+        });
+    });
+    describe('when called with no parameters', () => {
+        it('returns a bigint', async () => {
```

Comment:
I guess there are probably reasons why it's not safe to presume that the test validator will always return `0n` here? I seem to remember that being a problem before: https://github.com/solana-labs/solana/issues/23853